test/tests/functional/pbs_reliable_job_startup.py

```diff
@@ -5669,3 +5696,149 @@ def test_t21(self):
             job_out = fd.read()
 
         self.assertEquals(job_out, expected_out)
+
+    def test_t22(self):
```

Comment:
I was expecting to see a decorator that would skip this test if not on a Cray system, but it appears you are making the nodes look like Cray nodes by adjusting the vntype. What happens when this test is actually run on a Cray where the vntype is already set properly?

Reply:
The vntype setting set in qmgr will override what the cray pbs_mom has put in. So this test should run within a Cray environment and a non-Cray environment.

Reply:
Just surprised to see Cray focused functionality being tested on non-Cray systems. I don't believe we've ever done that.